src/chainparams.cpp

```diff
@@ -275,6 +267,7 @@ class CRegTestParams : public CChainParams {
         consensus.BIP34Hash = uint256();
         consensus.BIP65Height = 1351; // BIP65 activated on regtest (Used in functional tests)
         consensus.BIP66Height = 1251; // BIP66 activated on regtest (Used in functional tests)
+        consensus.CSVHeight = 432; // CSV activated on regtest (Used in rpc activation tests)
```

Comment:
hmmm, it's an interesting point, but I don't think there's an issue here. Previously, feature_csv_activation.py activated CSV at height 576, so that was still before bip 65/66 activation.